### PR TITLE
add documentation for add basic auth with traefik

### PR DESCRIPTION
**Pull request by elmalakomar**
## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensible content has been committed

### CODE REVIEW

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/how_to/traefik/traefik_handbook.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Add the middleware `default-basicauth` in the host where you want to setup the basic auth.
+Example:
+```
+routes:
+    - match: Host(`my-hostname.eu`)
+      middlewares:
+        ...
+        - name: "default-basicauth"
+```
+### Deploy
```
Comment (peppemanzi) [inline docs/how_to/traefik/traefik_handbook.mdx:81]: It's not a tutorial. I would remove the deploy chapter.

**Review by peppemanzi — CHANGES_REQUESTED** [review]

File: docs/how_to/traefik/traefik_handbook.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+id: traefik_handbook
+title: Traefik Handbook
+sidebar_label: Traefik Handbook
+---
+
+import SourceCodeBlock from "./../../../src/components/SourceCodeBlock";
+
```
[the remark this answers is not in the document]
Comment (elmalakomar) [reply docs/how_to/traefik/traefik_handbook.mdx:8]: added brief introduction and note box telling that guide is useful for infrastructure with Traefik as ingress

File: docs/how_to/traefik/traefik_handbook.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+Create a new secret with the basic auth credentials. The [Traefik documentation](https://doc.traefik.io/traefik/middlewares/http/basicauth/#general) suggest to use the `htpasswd` command to evaluate the secret.
+
+Example with username and password:
+```
+echo $(htpasswd -nb username password)
+```
+
+The output of the above command is the value of the secret to create.
+
+The secret needs to be created through one of the [supported environments variable provider](https://docs.mia-platform.eu/docs/development_suite/set-up-infrastructure/env-var).
```
[the remark this answers is not in the document]
Comment (elmalakomar) [reply docs/how_to/traefik/traefik_handbook.mdx]: changed the reference

File: docs/how_to/traefik/traefik_handbook.mdx (hunk 1 above)
Comment (elmalakomar) [reply docs/how_to/traefik/traefik_handbook.mdx]: added closing sentence

**Review by peppemanzi — APPROVED** [review]

**Pull request merged**